eslint.config.mjs

```diff
@@ -0,0 +1,104 @@
+import tseslint from 'typescript-eslint';
+import ddgConfig from "@duckduckgo/eslint-config";
+import globals from "globals";
+
+// @ts-check
+export default tseslint.config(
+    ...ddgConfig,
```

Comment:
Is this spreading not possible in an eslintrc? (that seems the only reason to change file format)

Reply:
Since we're updating to eslint 9, I wanted to use the flat config to avoid possible incompatibilities

Reply:
Should use `eslint.config.js` then?

I've got zero context on the new format just I'm really not a fan of config files written in dynamic languages.

Reply:
It's their default config format now, for better or for worse. It is dynamic, but the actual behaviour is much more simple now.